changes/v4.4.15-en.md

```diff
@@ -0,0 +1,7 @@
+
+# v4.4.15
+
+## Enhancements
+- The MongoDB library has been upgraded to support MongoDB version 5.1 and greater [#9707](https://github.com/emqx/emqx/pull/9707).
```

Comment:
Reminder: need to add ZH translation (a first approximation may be obtained with [deepl](https://www.deepl.com/translator)).

Reply:
Very cool. I added a translation from deepl. At least it made sense according to one Chinese non-technical person that I asked.